tests/react/vanilla-utils/atomWithObservable.test.tsx

```diff
@@ -408,7 +367,9 @@ it('synchronous subscription with BehaviorSubject', async () => {
 
   const { findByText } = render(
     <StrictMode>
-      <Counter />
+      <Suspense fallback="loading">
```

Comment:
Idk how, but tests were running perfectly fine locally. Only on CI where testing against React 17 will it trigger a suspense, hence needing a fallback :p